FlowCrypt/Controllers/Compose/ComposeViewController.swift

```diff
@@ -697,11 +697,11 @@ extension ComposeViewController: UIImagePickerControllerDelegate, UINavigationCo
             attachment = ComposeMessageAttachment(librarySourceMediaInfo: info)
         default: fatalError("No other image picker's sources should be used")
         }
-        guard let attachment = attachment else {
+        guard let att = attachment else {
             showAlert(message: "files_picking_photos_error_message".localized)
             return
         }
-        appendAttachmentIfAllowed(attachment)
+        appendAttachmentIfAllowed(att)
```

Comment:
Going forward I'd like to spell it out as `attachment` instead of `att` because I found that developers new to the project may often read it `att` as `attribute` and be confused (I even had someone rename it to `attribute` in another codebase).

Reply:
Got it. Just had some problems with compiling code